sagemaker/run/infer_entry.py

```diff
@@ -50,6 +50,8 @@ def parse_train_args():
     parser.add_argument("--custom-script", type=str, default=None,
         help="Custom training script provided by a customer to run customer training logic. \
             Please provide the path of the script within the docker image")
+    parser.add_argument("--output-chunk-size", type=int, default=100000,
+        help="Number of rows per chunked prediction result or node embedding file.")
```

Comment:
Do we need to do something simlar for do-nid-remap？

Reply:
No. We have predefined behavior: Training will with --do-nid-remap False and Inference will with --do-nid-remap True.